activities/Abecedarium.activity/buttons.js

```diff
@@ -40,6 +40,31 @@ enyo.kind({
 	// Constructor
 	rendered: function() {
 		this.inherited(arguments);
+		function hideLangandCasebutton(){
+			var x = document.getElementsByClassName("switchCase");
+			var y = document.getElementsByClassName("switchLang");
+			var i;
+			for (i = 0; i < x.length; i++) {
+				x[i].style.visibility = "hidden";
+			}
+			for (i = 0; i < y.length; i++) {
+				y[i].style.visibility = "hidden";
+			}
+		}
+		function showLangandCasebutton(){
+  			var x = document.getElementsByClassName("switchCase");
+  			var y = document.getElementsByClassName("switchLang");
+  			var i;
+  			for (i = 0; i < x.length; i++) {
+  				x[i].style.visibility = "visible";
+  			}
+  			for (i = 0; i < y.length; i++) {
+  				y[i].style.visibility = "visible";
+  			}
+  		}
+		if (document.getElementById("main-toolbar").style.visibility == "hidden")
+			hideLangandCasebutton()
+		document.getElementById("main-toolbar").style.visibility = "visible";
```

Comment:
Do you really need to do that? It's not a good thing to act on a global object in a component method.